client/src/client-only-routes/show-certification.tsx

```diff
@@ -41,6 +39,7 @@ import {
   certTypes,
   certTypeTitleMap
 } from '../../../shared/config/certification-settings';
+import MultiTierDonationForm from '../components/Donation/multi-tier-donation-form';
```

Comment:
DonateForm is replaced with MultiTierDonationForm. The multi-tier form will wrap the donate form only after tier selection